packages/zwave-js/src/lib/commandclass/SceneActuatorConfigurationCC.ts

```diff
@@ -305,25 +326,35 @@ export class SceneActuatorConfigurationCCReport extends SceneActuatorConfigurati
 		super(driver, options);
 		validatePayload(this.payload.length >= 3);
 		this.sceneId = this.payload[0];
-		this.level = this.payload[1];
-		this.dimmingDuration =
-			Duration.parseReport(this.payload[2]) ?? new Duration(0, "unknown");
 
-		this.persistValues();
+		if (this.sceneId != 0) {
+			this.level = this.payload[1];
+			this.dimmingDuration =
+				Duration.parseReport(this.payload[2]) ??
+				new Duration(0, "unknown");
+
+			this.persistValues();
+		}
 	}
 
-	public sceneId: number;
-	public level: number;
-	public dimmingDuration: Duration;
+	public readonly sceneId: number;
+	public readonly level?: number;
+	public readonly dimmingDuration?: Duration;
 
 	public persistValues(): boolean {
-		persistSceneActuatorConfig.call(
+		if (
+			this.sceneId === 0 ||
+			this.level == undefined ||
+			this.dimmingDuration == undefined
+		) {
+			return false;
+		}
```

Comment:
```suggestion
```
No need, you're not calling this if sceneId is 0.

Reply:
Okay, I added that because I wasn't sure if there was a possibility for `persistValues()` to be called from outside of my code.

If it can't be called from outside the class, why is it `public`?

Reply:
I ended up leaving in this code and instead moving the call to `persistValues()` out of the `sceneId === 0` check in the constructor.

This was mostly done to please the Lint gods, but I'm also wary of removing necessary checks from a public function (even if I know it should never be called with parameters that fail the check).